zanata-rest-client/src/main/java/org/zanata/rest/client/RestClientFactory.java

```diff
@@ -81,10 +81,12 @@ public RestClientFactory(URI base, String username, String apiKey,
         clientConfig.getClasses().add(JacksonJsonProvider.class);
 
         client = Client.create(clientConfig);
+        client.addFilter(new RedirectFilter());
```

Comment:
No this one has to be added first before other filter

On Tue, Mar 1, 2016 at 1:56 PM, Alex Eng <notifications@github.com> wrote:

> In
> zanata-rest-client/src/main/java/org/zanata/rest/client/RestClientFactory.java
> <https://github.com/zanata/zanata-client/pull/100#discussion_r54520026>:
>
> > @@ -81,10 +81,12 @@ public RestClientFactory(URI base, String username, String apiKey,
> >          clientConfig.getClasses().add(JacksonJsonProvider.class);
> >
> >          client = Client.create(clientConfig);
> > +        client.addFilter(new RedirectFilter());
>
> No big deal, but alphabetical would be useful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-client/pull/100/files#r54520026>.
>



-- 
Patrick Huang
Senior Software Engineer
Engineering - Internationalisation
Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8278
Fax: +61 7 3514 8199
IRC: pahuang
github: github.com/huangp
Website: www.redhat.com